whole-game.Rmd

```diff
@@ -187,9 +189,9 @@ In the *Files* pane, go to *More (gear symbol) \> Show Hidden Files* to toggle t
 A select few are visible all the time, but sometimes you want to see them all.
 :::
 
--   `.Rbuildignore` lists files that we need to have around but that should not be included when building the R package from source. More in @sec-rbuildignore.
+-   `.Rbuildignore` lists files that we need to have around but that should not be included when building the R package from source. By default, this file (as well as `.gitignore`) is only created by `create_package()` if you are using RStudio, however you will likely need to make use of it in the future regardless of what editor you are using. It is discussed in more detail in @sec-rbuildignore.
```

Comment:
I wasn't really conscious of this, but I did the experiment and see what you mean. Now that I get it, I'm going to take a crack at rewording.

```suggestion
-   `.Rbuildignore` lists files that we need to have around but that should not be included when building the R package from source. If you aren't using RStudio, `create_package()` may not create this file (nor `.gitignore`) at first, since there's no RStudio-related machinery that needs to be ignored. However, you will likely develop the need for `.Rbuildignore` at some point, regardless of what editor you are using. It is discussed in more detail in @sec-rbuildignore.
```

Reply:
I like that, it's clearer

Reply:
Forgive me if you did this or already know this, but just in case, you can accept suggestions like this by clicking on "Commit suggestion" in this PR interface here in the browser. Then it will give you a chance to commit right away or build up a batch of suggestions and commit as one. Then you'd want to pull these new changes into your local branchsd right away.

Reply:
Yup, I know and love that feature!